ui/js/component/fileActions/view.jsx

```diff
@@ -83,6 +119,40 @@ class FileActions extends React.PureComponent {
       title = metadata ? metadata.title : uri;
 
     let content;
+    let tipLink = (
+      <Link
+        label={__("Tip Publisher")}
+        button="text"
+        icon="icon-gift"
+        onClick={this.handleTipPublisherButtonClicked.bind(this)}
+      />
+    );
+
+    let tipBox = (
```

Comment:
Ok. I'll do it.
That's how it was originally done(the modal one), but I got confused with "add button to `fileActions`" part. A slight misinterpretation on my part.